starter-code/app.js

```diff
@@ -0,0 +1,18 @@
+// Require Express
+const express = require('express');
+
+// Express server handling requests and responses
+const app = express();
+
+// Make everything inside of public/ available
+app.use(express.static('public'));
+
+// explicity define home.html as "home page"
+app.get('/', (request, response, next) => {
```

Comment:
We expected you to put all your files under a `views` instead of putting them under the `public` one.
If your views are public, then everyone can see them without going through server code.